src/material/tooltip/tooltip.ts

```diff
@@ -543,6 +550,12 @@ export class MatTooltip implements OnDestroy, AfterViewInit {
 
   /** Binds the pointer events to the tooltip trigger. */
   private _setupPointerEvents() {
+    // Optimization: Defer hooking up events if there's no message or the tooltip is disabled.
+    if (this._disabled || !this.message || !this._viewInitialized ||
```

Comment:
I think it's safer to be here - otherwise it could accidentally be omitted. Would adding a return boolean value here be ok for if the caller needs to know?

Alternatively, they can just check this._passiveListeners.size

Reply:
This is a private API so we can probably get away with not returning what the result was. Let's keep it like this for now and we can rework it if it becomes an issue.

Reply:
I'm fine with this behavior with the names in your follow-up PR